packages/squiggle-lang/__tests__/TS/Scalars_test.ts

```diff
@@ -0,0 +1,51 @@
+import { errorValueToString } from "../../src/js/index";
+import { testRun } from "./TestHelpers";
+import * as fc from "fast-check";
+
+describe("Scalar manipulation is well-modeled by javascript math", () => {
```

Comment:
This is just testing mathjs stuff, not Distribution stuff. I don't think we want tests of mathjs math. (Though we could have tests to make sure that basic mathjs functionality isn't throwing errors)